qa/rpc-tests/mempool_tx_input_limit.py

```diff
@@ -106,20 +120,20 @@ def run_test(self):
         recipients = []
         spend_taddr_amount = spend_zaddr_amount - Decimal('0.0001')
         spend_taddr_output = Decimal('8')
+
         # Create three outputs
-        self.call_z_sendmany(node0_zaddr, node1_taddr, spend_taddr_output - Decimal('0.0001'))
-        self.nodes[1].generate(1)
-        self.sync_all()
-        self.call_z_sendmany(node0_zaddr, node1_taddr, spend_taddr_output - Decimal('0.0001'))
-        self.nodes[1].generate(1)
-        self.sync_all()
-        self.call_z_sendmany(node0_zaddr, node1_taddr, spend_taddr_amount - spend_taddr_output - spend_taddr_output - Decimal('0.0001')) # note amount less fees
+        recipients.append({"address":self.nodes[1].getnewaddress(), "amount": spend_taddr_output})
+        recipients.append({"address":self.nodes[1].getnewaddress(), "amount": spend_taddr_output})
+        recipients.append({"address":self.nodes[1].getnewaddress(), "amount": spend_taddr_amount - spend_taddr_output - spend_taddr_output})
```

Comment:
ACK; the first variant was left over from an earlier version where node 1 already had a balance, so I was trying to get everything into one t-addr, but couldn't make multiple TXOs in the same `z_sendmany`.